### PR TITLE
chore: remove Docker Hub and add Harbor to the adoption section

### PR DESCRIPTION
**Pull request by FeynmanZhou**
Docker Hub doesn't support the Notary Project signature so we need to remove it from the adoption section. Harbor recently supported Notation in v2.9.0, see the [reference](https://goharbor.io/blog/harbor-2.9/).

### CODE REVIEW

File: content/en/_index.html [hunk 1]
```diff
@@ -173,8 +173,7 @@ <h1 class="text-left">Multi-registry support</h1>
         <div class="description is-bullets">
           <ul>
             <li>
-              It supports push and store signatures alongside the artifacts in OCI registries, such as Docker Hub, ACR,
-              Zot registry, etc.
+              It supports push and store signatures alongside the artifacts in OCI registries
```
Comment (yizha1) [inline content/en/_index.html:176]:
```suggestion
              It supports pushing and storing signatures alongside the artifacts in OCI compliant registries
```

File: content/en/_index.html [hunk 2]
```diff
@@ -238,12 +237,11 @@ <h1 class="title">Adopted and trusted by</h1>
...
   <div class="horizontalCard gap-md space-between is-align-items-center flex">
     <div class="img-container-md">
-      <img src="docker_logo.svg" alt="Aqua logo" />
+      <img src="harbor_logo.svg" alt="Aqua logo" />
     </div>
     <div class="text-container text-left">
       <p>
-        Docker Hub supports signing container images with Notation and storing signatures and other supply chain
-        artifacts
+        Harbor supports signing container images with Notation and storing Notary Project signatures in the registry
```
Comment (yizha1) [inline content/en/_index.html:244]:
```suggestion
        Harbor supports storing Notary Project signatures alongside artifacts in the registry
```

**Review by yizha1 — COMMENTED** [review]
Thanks @FeynmanZhou

**Comment by zr-msft** [comment]
Agree with suggestions from @yizha1, othewise LGTM

**Review by yizha1 — APPROVED** [review]
LGTM

**Comment by FeynmanZhou** [comment]
@zr-msft Would you mind reviewing it again? Thanks

**Review by priteshbandi — APPROVED** [review]
LGTM

**Pull request merged**